dev_apps/uitest/js/HW/bluetooth.js

```diff
@@ -45,16 +46,49 @@ function bluetoothTest() {
 
   if (btHelper.v2) {
     version.textContent = "v2";
-    bluetooth.onadapteradded = function onAdapterAdded(evt) {
-      update();
-    }
+    bluetooth.addEventListener('attributechanged', (evt) => {
+      for (var i in evt.attrs) {
+        switch (evt.attrs[i]) {
+          case 'defaultAdapter':
+            var adapter = bluetooth.defaultAdapter;
+            if (adapter.address) { // for default On case
```

Comment:
the BT address will already got when BT already ON, so the 'attributechanged' wont get the event